<launch>
  <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=/image/rectified raw out:=/test" />
</launch>
